File: src/me/memorial/module/modules/combat/HitBox.java
package me.memorial.module.modules.combat;

import me.memorial.Memorial;
import me.memorial.module.Module;
import me.memorial.module.ModuleCategory;
import me.memorial.module.ModuleInfo;
import me.memorial.value.FloatValue;

@ModuleInfo(name = "HitBox", description = "Makes hitboxes of targets bigger.", category = ModuleCategory.COMBAT)
public class HitBox extends Module {
    public static HitBox instance;

    public HitBox() {
        instance = this;
    }

    public FloatValue sizeValue = new FloatValue("Size", 0.4F, 0F, 1F);
}

File: src/me/memorial/module/modules/combat/HitBox.kt
package me.memorial.module.modules.combat

import me.memorial.module.Module
import me.memorial.module.ModuleCategory
import me.memorial.module.ModuleInfo
import me.memorial.value.FloatValue

@ModuleInfo(name = "HitBox", description = "Makes hitboxes of targets bigger.", category = ModuleCategory.COMBAT)
class HitBox : Module() {
    companion object {
        var instance: HitBox? = null
    }

    init {
        instance = this
    }

    val sizeValue = FloatValue("Size", 0.4F, 0F, 1F)

}